commit 11460f0f764c54d68b4c3f6ab66b8d6ac7fda5e6
Author: Marc Kleine-Budde <mkl@pengutronix.de>
Date:   Thu Mar 2 12:03:40 2017 +0100

    can: usb_8dev: Fix memory leak of priv->cmd_msg_buffer
    
    commit 7c42631376306fb3f34d51fda546b50a9b6dd6ec upstream.
    
    The priv->cmd_msg_buffer is allocated in the probe function, but never
    kfree()ed. This patch converts the kzalloc() to resource-managed
    kzalloc.
    
    Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1305_11460f0f764c54d68b4c3f6ab66b8d6ac7fda5e6.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1305_11460f0f764c54d68b4c3f6ab66b8d6ac7fda5e6.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
